.travis.yml

```diff
@@ -9,19 +9,18 @@ services:
   - docker
 
 before_install:
-  - docker-compose up -d
+  - docker-compose -f docker-compose.yml up -d
```

Comment:
Does this really speed it up ?

Reply:
Nope, I tried to cache the docker builds but Travis CI recommend that you don't do that. So this PR just turned into cleaning  up the travis.yml file and adding some badges to the readme :)

Reply:
Ah cool. Makes sense